doc/admin-guide/files/records.config.en.rst

```diff
@@ -3361,6 +3361,10 @@ SSL Termination
    ``1`` Disable the SSL session cache for a connection during lock contention.
    ===== ======================================================================
 
+.. ts:cv:: CONFIG proxy.config.ssl.server.session_ticket_enable INT 1
```

Comment:
Should this be proxy.config.ssl.server.session_ticket.enable ?

Reply:
(just cause I read it 20 times without success to find the difference --- it's the _ or . between ticket and enable)

Reply:
Yep, that was an erroneous change.  Should be a '.' not a '_'.  Will revert that line.  Good eye test.